cli/ballerina-cli/src/main/resources/cli-help/ballerina-tool-pull.help

```diff
@@ -8,6 +8,9 @@ OPTIONS
        -h, --help
            Print the usage details of tool pull command.
 
+       --repository
+           Pull a tool from a custom repository.
```

Comment:
```suggestion
           Pull a tool from a custom repository.
```
@sameerajayasoma do we need to consider supporting custom repositories for publishing tools? If not, we can reword this to ` Pull a tool from a custom repository.`

Reply:
Let's change this to:
```suggestion
           Pull a tool from a local repository.
```